mne/io/base.py

```diff
@@ -1997,6 +2004,16 @@ def _write_raw(fname, raw, info, picks, fmt, data_type, reset_range, start,
 
         pos_prev = pos
 
+    if raw.annotations is not None:
+        start_block(fid, FIFF.FIFFB_MNE_ANNOTATIONS)
+        write_int(fid, FIFF.FIFF_MNE_BASELINE_MIN, raw.annotations.onset)
+        write_int(fid, FIFF.FIFF_MNE_BASELINE_MAX, raw.annotations.duration)
```

Comment:
(I know it's already a violation of the tag name since you're not writing a `baseline` exactly, but it's less of a violation than writing onset and duration)